management/src/main/scala/akka/management/AkkaManagement.scala

```diff
@@ -165,7 +165,7 @@ final class AkkaManagement(implicit system: ExtendedActorSystem) extends Extensi
     if (binding == null) {
       Future.successful(Done)
     } else if (bindingFuture.compareAndSet(binding, null)) {
-      val stopFuture = binding.flatMap(_.unbind()).map(_ => Done)
+      val stopFuture = binding.flatMap(_.unbind())
```

Comment:
Is this the place that triggered the `java.lang.ClassCastException: akka.Done$ cannot be cast to scala.runtime.BoxedUnit`? That's a bit of a bummer... perhaps we should change it to `.map{ (_: Object) => Done }` to be on the safe side?

Reply:
>Is this the place that triggered the `java.lang.ClassCastException: akka.Done$ cannot be cast to scala.runtime.BoxedUnit`?

Yes it is!

>perhaps we should change it to .map{ (_: Object) => Done } to be on the safe side?

Yeah I hope `(_ : Object)` potentially could solve the issue for `10.0.x` and for `10.1.x` we can just remove the whole `.map` part as I did.

Reply:
This is because of the binary incompatible change in https://github.com/akka/akka-http/pull/1908

Reply:
That's indeed a bummer and we should make the change as hinted above to make it compatible with 10.0 if we make another release before updating to akka-http 10.1.x.

Reply:
Done: https://github.com/akka/akka-management/pull/280

Reply:
Ah, cool. What we can also do it akka-http is just returning `null` instead of `Done` to rule out problems with third-party code.